help/moving-to-zulip.md

```diff
@@ -0,0 +1,207 @@
+   discounted plan, you can [apply for
+   sponsorship](/help/self-hosted-billing#apply-for-community-plan). To inquire
+   about Zulip Enterprise, please reach out to
+   [sales@zulip.com](mailto:sales@zulip.com).
+
+{end_tabs}
+
+## Configure your organization
+
+{start_tabs}
+
+1. [Create your organization profile](/help/create-your-organization-profile),
+   which is displayed on your organization's registration and login pages.
+
+1. Review [organization permissions](/help/roles-and-permissions), such as who
+   can invite users, create channels, etc.
+
+1. If your organization uses an issue tracker (e.g., GitHub, Salesforce,
+   Zendesk, Jira, etc.), configure [linkifiers](/help/add-a-custom-linkifier) to
+   automatically turn issue numbers (e.g., #2468) into links.
+
+1. Set up [custom profile fields](/help/custom-profile-fields), which make it
+   easy for users to share information, such as their pronouns, job title, or
+   team.
+
+1. Review [default user settings](/help/configure-default-new-user-settings),
+   including language, [default visibility for email
+   addresses](/help/configure-email-visibility), and notification preferences.
+
+1. [Create channels](/help/create-channels), unless you've imported
+   channels from another app. Zulip's [topics](/help/introduction-to-topics)
+   give each conversation its own space, so one channel per team should be
+   enough to get you started.
+
+1. *(optional)* [Set up integrations](/help/set-up-integrations) so that your
+   team can experience all their regular workflows inside the Zulip app. Zulip's
+   [Slack-compatible incoming
+   webhook](https://zulip.com/integrations/doc/slack_incoming) makes it easy to
+   move your integrations when migrating an organization from Slack to Zulip.
+
+{end_tabs}
+
+## Prepare users for the transition
+
+{start_tabs}
+
+1. Plan how you will introduce users to Zulip. You may want to:
+
+    - Share Zulip's [getting started guide](/help/getting-started-with-zulip).
+    - Prepare a live demo / training session. Consider recording it for
+     future use.
+
+1. Inform users about the transition, including reasons for the change, the
+   timeline, and what they'll need to do.
+
+{end_tabs}
+
+## Invite users to join
+
+{!how-to-invite-users-to-join.md!}
+
+## Update your guidelines
+
+{start_tabs}
+
+1. Update any links and login instructions to point to your Zulip organization.
+
+1. Share basic instructions for getting started with Zulip. You can refer users
+   to Zulip's [help center](/help), [getting started
+   guide](/help/getting-started-with-zulip), and any onboarding content you've
+   created.
+
+1. Consider updating your organization's communication policies and best
+   practice recommendations to take advantage of Zulip's organized
+   conversations:
+
+    - Many organizations find that with Zulip, there’s no longer a reason to use
+      email internally. You get the organization of an email
```

Comment:
"email internally" => "email for internal communications"? Just seems helpful to be specific; there will probably be some automated systems that use email.